utils/storage_utils.py

```diff
@@ -60,19 +62,26 @@ def write_output_csv(start_time, end_time, cve_model_type, ecosystem, df, s3_upl
         "body"
     ]
     df = df[columns]
-    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, cc.FULL_OUTPUT)
+    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, oc.FULL_OUTPUT)
 
     # Now save the probable securities dataset.
     df = df[df.security_model_flag == 1].drop(["triage_is_cve"], axis=1)
-    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, cc.PROBABLE_SECURITY_AND_CVES)
+    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, oc.PROBABLE_SECURITY_AND_CVES)
 
     # Now save the probable CVE dataset.
     df = df[df.cve_model_flag == 1].drop(["triage_is_security"], axis=1)
-    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, cc.PROBABLE_CVES)
+    save_data_to_csv(df, s3_upload, file_prefix, new_triage_subdir, ecosystem, oc.PROBABLE_CVES)
 
     return df
 
 
+def _restrict_data_length(data: str):
+    """Ristrict long length data content."""
+    if data and len(data) > oc.STRING_CHAR_LIMIT:
+        return data[0:oc.STRING_CHAR_LIMIT]
+    return data
```

Comment:
No need to check for a condition. 
```suggestion
        data[0:oc.STRING_CHAR_LIMIT]
```

Reply:
Thanks for suggestion, removed condition now .